pkg/services/sqlstore/searchstore/builder.go

```diff
@@ -115,15 +115,22 @@ func (b *Builder) applyFilters() (ordering string) {
 		b.params = append(b.params, whereParams...)
 	}
 
-	if len(groups) > 0 {
-		b.sql.WriteString(fmt.Sprintf(" GROUP BY %s", strings.Join(groups, ", ")))
-		b.params = append(b.params, groupParams...)
-	}
-
 	if len(orders) < 1 {
 		orders = append(orders, TitleSorter{}.OrderBy())
 	}
 
+	if len(groups) > 0 {
+		cols := make([]string, 0, len(orders)+len(groups))
+		for _, o := range orders {
+			o := strings.TrimSuffix(o, " DESC")
+			o = strings.TrimSuffix(o, " ASC")
+			cols = append(cols, o)
+		}
+		cols = append(cols, groups...)
```

Comment:
Could it be that an ordered column is already in `groups`?

Reply:
good point! I believe that this wouldn't affect the query result but it may have performance implications, therefore better to filter them out, thanks!